ERP_INES/Domain/Modules/Finance/DTOs/PaymentMethodDto.cs

```diff
@@ -0,0 +1,10 @@
+namespace ERP_INES.Domain.Modules.Finance.DTOs;
+
+public class PaymentMethodDto
+{
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public string? Description { get; set; }
+    public DateTime? CreatedAt { get; set; }
```

Comment:
created at and updated at should be provided by the controller not for the user input